src/runtime/contrib/cudnn/conv_forward.cc

```diff
@@ -287,6 +355,91 @@ TVM_REGISTER_GLOBAL("tvm.contrib.cudnn.conv2d.find_algo")
   }
 
   ret[0] = best_algo;
+}
+
+
+TVM_REGISTER_GLOBAL("tvm.contrib.cudnn.conv.forward")
+.set_body([](TVMArgs args, TVMRetValue *ret) {
+  int mode = args[0];
+  int format = args[1];
+  int algo = args[2];
+  int dims = args[3];
+
+  Array<Expr> pads =  args[4];
+  Array<Expr> strides =  args[5];
+  Array<Expr> dilations =  args[6];
+
```

Comment:
This seems wrong... Expr and Array are compile-time stuff, they should not appear in the runtime code.
Ideally they should be void or int pointers, but I don't know how to make it work.

Reply:
How about form a string on python and parse it in C? But only if we don't have a better approach?

Reply:
No that sounds bad. In the worst case we need to go back to explicitly passing all values (i.e. we need to have separate conv2d_forward and conv3d_forward)